eng/pipelines/templates/steps/verify-autorest.yml

```diff
@@ -0,0 +1,40 @@
+parameters:
+  - name: ServiceToTest
+    type: string
+    default: ''
+  - name: SDKType
+    type: string
+    default: 'client'
+  - name: ServiceDirectory
+    type: string
+
+variables:
+  NodeVersion: '12.x'
+  PythonVersion: '3.9'
+  auto_rest_clone_url: 'https://github.com/Azure/autorest.python.git'
+
+steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(PythonVersion)'
+
+  - task: NodeTool@0
+    displayName: 'Install Node.js'
+    inputs:
+      versionSpec: '$(NodeVersion_'
```

Comment:
not accessing variable properly 

(sorry I know it's draft but it jumped out at me lol)